### DISCUSSION TITLE
Multiple time series with one alarm

### DISCUSSION THREAD

**Discussion by carvjust** [grafana/grafana#42168] [Alerting]
One of our customers has a query that returns multiple time series. The customer wants to set a single alarm that will alert them to all time series report no data. This functionality does not seem to be supported currently by Grafana.

**Comment by peterholmberg** [comment]
Could https://github.com/grafana/grafana/pull/40540 solve this?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
